receiver/sqlserverreceiver/README.md

```diff
@@ -36,6 +36,20 @@ Windows-specific options:
 - `computer_name` (optional): The computer name identifies the SQL Server name or IP address of the computer being monitored.
   If specified, `instance_name` is also required to be defined. This option is ignored in non-Windows environments.
 
+Query-level metric specific options (only useful when query-level metrics collection are enabled):
+- `lookback_time` (optional, example = `60`, default = `10`): The time window (in second) in which to query for metrics.
+- `max_query_sample_count` (optional, example = `5000`, default = `10000`): The maximum number of records to fetch in a single query.
+- `top_query_count`: (optional, example = `100`, default = `200`): The maximum number of record to report (to the next consumer) in a single query.
```

Comment:
better document it should not exceed 200

Reply:
@XSAM suggested to remove the 200 limitation and it makes sense. As we don't actually have the limitation in the code, I think it should be find to have a larger top_query_count configured.